src/effects/lv2/LV2Effect.cpp

```diff
@@ -835,8 +835,9 @@ bool LV2Effect::BuildFancy(LV2Validator &validator,
 
    const auto uinode = lilv_ui_get_uri(ui);
    lilv_world_load_resource(gWorld, uinode);
-   auto &features =
-      validator.mUIFeatures.emplace(mFeatures, (UIHandler&)*this, uinode);
+   auto &features = validator.mUIFeatures.emplace(
+      mFeatures, (UIHandler&)*this, uinode, &instance,
```

Comment:
I understand, that it is not a goal, but C-cast?

Reply:
That is a cast to base class, up-cast not down-, and needed because of private inheritance of `UIHander`.  Else the up-cast is attempted inside a function of `std::optional` where there isn't access.